apps/storybook.namekit.io/stories/Namekit/Tooltip.stories.tsx

```diff
@@ -5,18 +5,44 @@ import { Tooltip } from "@namehash/namekit-react/client";
 const meta: Meta<typeof Tooltip> = {
   title: "UI/Tooltip",
   component: Tooltip,
+  parameters: {
+    layout: "centered",
+  },
   argTypes: {
-    placement: {
-      options: ["top", "right", "bottom", "left"],
+    trigger: {
       control: { type: "select" },
+      options: ["Tooltip Trigger", "Any JSX element can be used as a trigger"],
+      mapping: {
+        "Tooltip Trigger": <>Tooltip Trigger</>,
+        "Any JSX element can be used as a trigger": (
+          <button disabled={true}>
+            Any JSX element can be used as a trigger
+          </button>
+        ),
+      },
     },
     children: {
-      control: { type: "text" },
+      control: { type: "select" },
+      options: ["Tooltip Trigger", "Any JSX element can be used as a trigger"],
+      mapping: {
+        "Tooltip content": <>Tooltip content</>,
+        "Any JSX element can be used as a content": (
+          <h1>Any JSX element can be used as a content</h1>
+        ),
+      },
     },
-    trigger: {
-      control: { type: "text" },
+    placement: {
+      options: ["top", "right", "bottom", "left"],
+      control: { type: "select" },
+    },
+    maxTooltipWidth: {
+      control: { type: "number" },
     },
   },
+  args: {
+    placement: "top",
+    maxTooltipWidth: 400,
```

Comment:
Why are you hardcoding `400` here? This kind of work creates all kinds of problems for maintainability. You should be using the constant we defined.